### DISCUSSION TITLE
The type initializer for 'NativeMagickSettings' threw an exception

### DISCUSSION THREAD

**Discussion by torojuro** [dlemstra/Magick.NET#1213] [Help]
I use ImageMagick quite often on Linux, since it's a very powerful tool when it comes to image manipulation tasks.
So I decided to use this Magick.NET wrapper in a VB.NET application I'm writing, but I'm facing an issue that someone else already had here and that seems to be more complicated then I thought.
I created a temporary Visual Studio 2008 project to test the library and see what it can do.
In order to import the library I downloaded:

- Magick.NET-Q16-x86.dll (.NET Framework 2.0)
- Magick.NET.Core.dll

After that, I imported the DLLs in the project and everything was fine, I was able to get Intellisense suggestions on the library objects and methods. Then I wrote this line of code

`Dim tmpImg As New ImageMagick.MagickImage(imgPath)`

and the application threw this exception: 

> The type initializer for 'NativeMagickSettings' threw an exception.

Looking at the inner exception I found this:

> Unable to load DLL 'Magick.NET-Q16-x86.Native.dll': The specified module could not be found. (HRESULT: 0x8007007E)

At this point I started digging on the web, to see if anyone encountered this thing before me and I found [this question](https://github.com/dlemstra/Magick.NET/issues/366) and several others.
Scrolling through the thread, I read that in that case was only a matter of adding a `System.Drawing` reference to the project to make the exception go away, but in my situation it didn't solve the problem.
I also granted full access to the application and compiled it for x86. I didn't copy the DLLs into `C:\Windows\System32` and `C:\Windows\SysWOW64` directories, because I don't want to make system wide alteration to only test the feature of a library and because it doesn't make any sense since I already have those libraries copied in the application _Debug_ folder and imported into the project.
As I wrote at the beginning, I am using Visual Studio 2008 Professional on a Windows 10 Pro (21H2) machine.

What am I missing?

**Comment by dlemstra** [comment]
It looks like `Magick.NET-Q16-x86.Native.dll` is not copied from the NuGet package to the output folder. This is probably happening because you are using an ancient version of Visual Studio.

p.s. Support for .NET Framework 2.0 will be removed in August with version 12.0.0.
